vagrant/provisioning/roles/haproxy/tasks/main.yml

```diff
@@ -438,14 +446,14 @@
   become: yes
   systemd:
     daemon_reload: yes
-  when: haproxy_service_name is changed
-
-- name: enable haproxy to connect to any port
-  become: yes
-  seboolean:
-    name: haproxy_connect_any
-    state: yes
-    persistent: yes
+  when: haproxy_serivce_name is changed
```

Comment:
Here you introduced a typo; it should remain haproxy_service_name versus haproxy_serivce_name